src/index.js

```diff
@@ -571,6 +574,10 @@ function createConfig(options, entry, format, writeMeta) {
 									target: 'esnext',
 								},
 							},
+							include: ['*.ts+(|x)', '**/*.ts+(|x)'].concat(
```

Comment:
not needed:
- default TS adds `*.ts,.*tsx` to the glob
- setting `allowJs` adds `*.js,*.jsx` to the glob